doc/sphinx/user_manual/concepts/boundless_world.md

```diff
@@ -2,22 +2,22 @@
+Users of the World Builder provide the locations where they want to have some tectonic features. They do this through providing map view coordinates (x and y) through the input file. We will explain later how we get the z coordinate. Coordinates can be set anywhere in an infinite 2D plane e.g., the origin (0,0) is not restricted to the "center" of the domain but all features share a common coordinate system. This is not an issue for the World Builder since when World Builder computes a property, e.g., temperature, at a location, it just checks whether a feature is present at that location. So no need to limit the domain! 
 
-The reason the GWB doesn't need a mesh is similar. When the world builder computes the temperature at a location, and it found that it is within the feature, then it can look where it is within that feature. That allows the temperature model to provide different temperatures based on where it is within the feature. 
+The reason the GWB doesn't need a mesh is similar. When the World Builder computes the temperature at a location, it determines whether it is within a feature and then determines its location within the feature. That allows the temperature model to provide different temperatures based on where it is within the feature. 
 
 ## Boundless spherical coordinate system
 
-When we are using a spherical coordinate system, the physical domain is limited to a sphere. The input in the world builder is not restricted by 360 degrees though. The reason for this is that you may want to cross the 180 or 360 barrier one or many times. To be able to do this 0 and 360 degrees need to be different coordinates, even though they may represent the same physical location. An example of what you can make with this can be seen below.
+When we are using a spherical coordinate system, the physical domain is limited to a sphere. The input in the World Builder is, however, not restricted to 360&deg; since you may want to cross the 180&deg; or 360&deg; barrier one or many times. To be able to do this, 0&deg; and 360&deg;need to be different coordinates even though they may represent the same physical location. An example of what you can make with this can be seen below.
 
 ![feature wrapping around sphere](https://user-images.githubusercontent.com/7631629/136886253-73ac05f5-4df0-420b-9bc3-dce7c53dd14e.png)
 
 ```{note}
-Although it should be possible to do more than the -380 to 380 range, only that range has been tested. If you are using a bigger range, please check the results carefully and report any issues.
+Although it should be possible exceed the -380&deg; to 380&deg; range, this has not been tested. If you are using a bigger range, please check the results carefully and report any issues.
```

Comment:
-360 to 360.

Reply:
See https://github.com/GeodynamicWorldBuilder/WorldBuilder/issues/338#issuecomment-940623565